### PR TITLE
Modify errorcode cannot be exported and cannot be imported correctly

### PR DESCRIPTION
**Pull request by qiudaoke**

### CODE REVIEW

**Comment by qiudaoke** [comment]
> Thanks for the contribution.
> 
> Can you please check the import statements especially in `ErrorsConverterTest` where the same class is imported twice? Also the project standards have static imports first in the file. If you are using Intellij there is a `idea-code-style-configuration.xml` file in the `ide-settings/idea` directory that can be used to help enforce the project code style.
> 
> Thanks.

ok ,I am already done it.

File: modules/flowable-ui-modeler/flowable-ui-modeler-logic/src/main/resources/stencilset_bpmn.json [hunk 1]
```diff
@@ -1165,7 +1165,17 @@
       "description" : "Message definitions",
       "popular" : true
     } ]
-  }, {
+  },{
+      "name" : "errordefinitionspackage",
+      "properties" : [ {
+        "id" : "errordefinitions",
+        "type" : "multiplecomplex",
+        "title" : "error definitions",
```
Comment (martin-grofcik) [inline modules/flowable-ui-modeler/flowable-ui-modeler-logic/src/main/resources/stencilset_bpmn.json]: Use upper case `Error` instead `error`

File: modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ErrorConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    protected String getResource() {
+        return "test.errormodel.json";
+    }
+
+    private void validateModel(BpmnModel model) {
+        FlowElement flowElement = model.getMainProcess().getFlowElement("sid-CF7571C4-F093-45C5-ADD6-8F357ACF584C");
+        assertNotNull(flowElement);
+        assertTrue(flowElement instanceof StartEvent);
+
+        StartEvent startEvent= (StartEvent) flowElement;
```
Comment (martin-grofcik) [inline modules/flowable-json-converter/src/test/java/org/flowable/editor/language/ErrorConverterTest.java]: Could you add error definitions to end and boundary event too?

File: modules/flowable-bpmn-converter/src/test/resources/errors.bpmn20.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <documentation>test_errorcode</documentation>
+    <startEvent id="sid-DC9C1AEF-C999-40B9-BAC5-6532CC6D7F89" isInterrupting="true">
+      <errorEventDefinition errorRef="shareniu-b"></errorEventDefinition>
+    </startEvent>
+    <userTask id="sid-58FAD9C6-DC70-4D28-9994-317CFC77969E"></userTask>
+    <sequenceFlow id="sid-955FBEA1-E4A3-4A73-9506-3B1E69EFB3AA" sourceRef="sid-DC9C1AEF-C999-40B9-BAC5-6532CC6D7F89" targetRef="sid-58FAD9C6-DC70-4D28-9994-317CFC77969E"></sequenceFlow>
+    <endEvent id="sid-06B9C63D-E14C-4DCA-8CC7-F6DA6CA4A8B2"></endEvent>
+    <sequenceFlow id="sid-3C8E5527-9DC5-44A4-83E7-3776F681C9BE" sourceRef="sid-58FAD9C6-DC70-4D28-9994-317CFC77969E" targetRef="sid-06B9C63D-E14C-4DCA-8CC7-F6DA6CA4A8B2"></sequenceFlow>
+  </process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_test_errorcode">
```
Comment (martin-grofcik) [inline modules/flowable-bpmn-converter/src/test/resources/errors.bpmn20.xml]: Diagram can be omitted.

File: modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java [hunk 4]
```diff
@@ -518,6 +518,9 @@ protected void addEventProperties(Event event, ObjectNode propertiesNode) {
                 TerminateEventDefinition terminateEventDefinition = (TerminateEventDefinition) eventDefinition;
                 propertiesNode.put(PROPERTY_TERMINATE_ALL, terminateEventDefinition.isTerminateAll());
                 propertiesNode.put(PROPERTY_TERMINATE_MULTI_INSTANCE, terminateEventDefinition.isTerminateMultiInstance());
+            }else if (eventDefinition instanceof ErrorEventDefinition) {
```
Comment (martin-grofcik) [inline modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java]: }else missing space

**Comment by tijsrademakers** [comment]
Closing the PR due to lack of activity, please reopen when you have time to work on it.

**Pull request closed without merging**